skills/declarative/calendarSkill/dialogs/SearchEventsDialog/SearchEventsDialog.dialog

```diff
@@ -108,7 +112,8 @@
           "isFutureProperty": "=coalesce(dialog.futureEventsOnly, false)",
           "isFutureEventsOnlyProperty": "=dialog.futureEventsOnly",
           "dateTimeTypeProperty": "= coalesce(dialog.type, \"date\")",
-          "startProperty": "=coalesce(dialog.start, formatDateTime(utcNow(), 'yyyy-MM-dd'))"
+          "startProperty": "=coalesce(dialog.start, formatDateTime(addDays(utcNow(),-1), 'yyyy-MM-dd'))",
```

Comment:
Why do you need to subtract from the current date? Won't this set the default date to "yesterday"?

Reply:
Actually this is a workaround, we should convert the utcNow to User's timezone because the 'dialog.start' is the start time from user. Directly using utcNow() will cause some error on certain conditions. Let me try to directly convert it to User's timezone.

Reply:
Ah, got it. You should be able to use user.workingHours.timeZone.name to do the conversion